src/site/filters/liquid.js

```diff
@@ -441,16 +441,24 @@ module.exports = function registerFilters() {
     return null;
   };
 
+  // TODO - add unit tests for this!
```

Comment:
Looks like a test was added. Does this still apply?

Reply:
not anymore, will remove